plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -43,6 +46,16 @@ ${project.additionalConfiguration}
         return dir
     }
 
+    void createGradleSettings() {
+        write("", 'settings.gradle')
+    }
+
+    void createGradleProperties() {
+        write("""
+org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
isn't 2gb too much? These are almost empty projects.

Reply:
Probably, however, `xmx` referrers to maximum memory, so should be ok. Happy to lower it though.

Reply:
I was just thinking this might be another reason of inflated execution time. It seems like it isn't.